log.ml

```diff
@@ -63,6 +63,19 @@ module State = struct
       Hashtbl.iter (fun k x -> if Stre.starts_with k prefix then Logger.set_filter x level) all
     | Some name -> Logger.set_filter (facility name) level
 
+  let mute' name mute =
+    let mute = if mute then Logger.mute else Logger.unmute in
+    match name with
+    | name when Stre.ends_with name "*" ->
+      let prefix = String.slice ~last:(-1) name in
+      Hashtbl.iter (fun k x -> if Stre.starts_with k prefix then mute x) all
+    | name -> mute (facility name)
+
+  let mute name = mute' name true
```

Comment:
instead of `mute` can have another loglevel call it Nothing, do not allow logging with it but allow using it as filter level. This will also work nicely to integrate with commandline (no need to add special support for mute)